src/plugins/unified_search/public/query_string_input/query_bar_menu.tsx

```diff
@@ -34,6 +34,10 @@ export const strings = {
     i18n.translate('unifiedSearch.filter.options.filterSetButtonLabel', {
       defaultMessage: 'Saved query menu',
```

Comment:
Is this the tooltip on for the menu title?  It should be "Query menu"

Reply:
Done https://github.com/elastic/kibana/pull/155238/commits/c096118f38211a036a84cd9517514ac445437f5e